vfio/src/vfio_pci.rs

```diff
@@ -894,9 +896,21 @@ impl PciDevice for VfioPciDevice {
             return 0;
         }
 
+        // Since we don't support INTx (only MSI and MSI-X), we should not
+        // expose an invalid Interrupt Pin to the guest. By using a specific
+        // mask in case the register being read correspond to the interrupt
+        // register, this code makes sure to always expose an Interrupt Pin
+        // value of 0, which stands for no interrupt pin support.
+        let mask = if reg_idx == PCI_INTX_REG_INDEX {
+            0xffff_00ff
+        } else {
+            0xffff_ffff
+        };
+
         // The config register read comes from the VFIO device itself.
         self.vfio_pci_configuration
             .read_config_dword((reg_idx * 4) as u32)
+            & mask
```

Comment:
Weird that cargo fmt let this extra space get through?

Reply:
It should be fixed as well.